lib/livebook/session/data.ex

```diff
@@ -877,7 +890,9 @@ defmodule Livebook.Session.Data do
       revision_by_client_pid: Map.new(client_pids, &{&1, 0}),
       validity_status: :fresh,
       evaluation_status: :ready,
-      evaluation_digest: nil
+      evaluation_digest: nil,
+      evaluation_start_time: 0,
+      evaluation_time_ms: 0
```

Comment:
I think we should keep it `nil` similarly to `evaluation_digest` for consistency.